pages.hi/common/babel.md

```diff
@@ -0,0 +1,37 @@
+# babel
+
+> एक ट्रांसपिलर जो कोड को जावास्क्रिप्ट ES6/ES7 सिंटैक्स से ES5 सिंटैक्स में परिवर्तित करता है।
+> एक ट्रांसपिलर जो कोड को जावास्क्रिप्ट ES6/ES7 सिंटैक्स से ES5 सिंटैक्स में परिवर्तित करता है।
+> अधिक जानकारी: <https://babeljs.io/>.
+
+- एक निर्दिष्ट इनपुट फ़ाइल और आउटपुट को stdout में ट्रांसपाइल करें:
+
+`babel {{path/to/file}}`
```

Comment:
```suggestion
`babel {{फ़ाइल/का/पथ}}`
```

Reply:
@kbdharun can you translate the other part? i could only change this since is standard